packages/angola/package.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "@svg-maps/angola",
+	"version": "1.0.1",
+	"description": "Map of Angola",
+	"main": "index.js",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/VictorCazanave/svg-maps/tree/master/packages/angola"
+	},
+	"keywords": [
+		"svg",
+		"map",
+		"angola",
+		"ao",
+		"aoa"
```

Comment:
Could you only list `"svg", "map", "angola"` to be consistent with the other maps please?